index.html

```diff
@@ -78,6 +84,181 @@ <h4>I am a Software Developer</h4>
           </ul>
         </div>
       </section>
+
+      <section class="content-container">
```

Comment:
[OPTIONAL]
- [ ] I suggest you use the Grid option on the whole work area section, and use flexbox for each individual work 😄